docs/usage/sat_and_iuf.md

```diff
@@ -0,0 +1,84 @@
+# SAT and IUF
+
+The Install and Upgrade Framework (IUF) provides commands which install,
+upgrade, and deploy products on systems managed by CSM with the help of
+`sat bootprep`. Outside of IUF, you will rarely need to use `sat bootprep`.
```

Comment:
I would maybe weaken this statement just a bit. Perhaps "uncommon" instead of "rare". And maybe remove "you".

```suggestion
`sat bootprep`. It is uncommon to use `sat bootprep` outside the context
of the IUF.
```

Reply:
Changing in the next commit to "Outside of IUF, it is uncommon to use `sat bootprep`."